ui-v2/app/components/consul-instance-checks/index.js

```diff
@@ -0,0 +1,52 @@
+import Component from '@ember/component';
+import { computed, get } from '@ember/object';
+
+export default Component.extend({
+  tagName: '',
+  healthCheck: computed(function() {
```

Comment:
We need to make sure we add any dependency properties as a first argument here, so in this case you'd need to add `items.[]` which will cover any changes to the `items` array.